### ISSUE TITLE
chore(ci): add the state tests generated by evmone's unittests to EEST releases

### ISSUE THREAD

**Issue by danceratopz** [ethereum/execution-specs#1469]
https://github.com/ethereum/evmone/pull/743 added state test fixture generation from evmone's unit tests. We could consider generating these state test fixtures as part of our fixture release CI flow to make them available to other teams.

To generate the fixtures, build evmone and then
```
export EVMONE_EXPORT_TESTS=exported_tests
build/bin/evmone-unittests --gtest_filter=state_transition.\*
ls exported_tests/state_transition/
# -> 
# block  create  eof  selfdestruct  trace  transient_storage  tx
```

**Comment by danceratopz** [comment]
Hi @chfast, this issue will be made redundant by your new issue?
- ethereum/execution-specs#1470

**Comment by chfast** [comment]
Not exactly, the ethereum/execution-specs#1470 is a list of test cases that are definitively missing in EEST.

This issue was about import evmone's unit tests in masses. I'm not sure you still want to do this.
